R/post-observer.R

```diff
@@ -105,10 +125,10 @@ make_gbm_predictions = function(x, use_obs_model){
     mean = mean + test$observer_effect
   }
   
-  cbind(test, mean = mean, model = "richness_gbm",  use_obs_model = use_obs_model, 
+  cbind(test, mean = mean, model = "richness_gbm", 
         stringsAsFactors = FALSE) %>% 
-    select(site_id, year, mean, richness, model, use_obs_model) %>% 
-    mutate(sd = sd)
+    select(site_id, year, mean, richness, model) %>% 
+    mutate(sd = !!sd, n.trees = !!n.trees, use_obs_model = !!use_obs_model)
```

Comment:
I think that your understanding is correct, and they're both a few lines above my diff (see below). 

I could probably name `sd` better to avoid a name clash with the existing `sd` column. The `!!` for `n.trees` isn't necessary because there isn't (currently) a column with that name, but I like to be explicit.

https://github.com/davharris/bbs-forecasting/blob/b3344f6da1a50faec8647c85795ce1b2c1e9287d/R/post-observer.R#L118-L121